<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: 'green';</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="dealer_card_label">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>50</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="player_card_label">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>50</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="shuffle_button">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>290</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'light gray'</string>
    </property>
    <property name="text">
     <string>Relancer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deal_button">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>290</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'light gray'</string>
    </property>
    <property name="text">
     <string>Poser</string>
    </property>
   </widget>
   <widget class="QLabel" name="dealer_label">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distributeur</string>
    </property>
   </widget>
   <widget class="QLabel" name="player_label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Joueur</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
